usaspending_api/download/management/commands/download_sqs_worker.py

```diff
@@ -101,7 +102,11 @@ def download_service_app(download_job_id):
 
 def _retrieve_download_job_from_db(download_job_id):
     download_job = DownloadJob.objects.filter(download_job_id=download_job_id).first()
-
+    limit = download_job.json_request.get("limit")
+    if limit is not None and limit > settings.MAX_DOWNLOAD_SIZE:
+        raise Exception(
+            f"Sorry, we cannot process this download because it includes more than the current limit of {settings.MAX_DOWNLOAD_SIZE} records"
+        )
```

Comment:
Would this be better after line 90?
```
download_job = _retrieve_download_job_from_db(download_job_id)
```

`_retrieve_download_job_from_db(download_job_id)` is used in other places, and we really just want to perform the validation before the `generate_download()` method is called right?